core/lib/Thelia/Controller/Admin/HomeController.php

```diff
@@ -74,61 +86,100 @@ public function loadStatsAjaxAction()
             return $response;
         }
 
-        $data = new \stdClass();
+        $cacheEnable = intval(ConfigQuery::read("admin_cache_home_stats_enable", '1'));
```

Comment:
I see you use a lot the config table in your PR and it's a good idea.

It's a good practice to create an alias when you want to retrieve the value of a specific config. Doing this you prevent typo and you can easily change the name of your variable without refactoring all your code. Moreover, each time you cast the result in int. Do this in the alias method.

++